### PR TITLE
Update arsc library with more functionality for mutation

### PR DESCRIPTION
**Pull request by ricowind**
This is explicitly not updating the binary entrypoints and the injection code

### CODE REVIEW

**Comment by ricowind** [comment]
@mkj-gram

File: java/com/google/devrel/gmscore/tools/apk/arsc/Chunk.java [hunk 1]
```diff
@@ -17,24 +17,25 @@
...
 import java.util.Map;
-
 import javax.annotation.Nullable;
 
 /** Represents a generic chunk. */
 public abstract class Chunk implements SerializableResource {
 
   /** Types of chunks that can exist. */
+  // See
+  // https://cs.corp.google.com/aosp-master/frameworks/base/libs/androidfw/include/androidfw/ResourceTypes.h?rcl=2b1b1bb22dcab2467f8a41825f36d6be68a031f5&l=215
```
Comment (wojtek-kalicinski) [inline java/com/google/devrel/gmscore/tools/apk/arsc/Chunk.java]: please scrub internal link or convert it to an external one if applicable

**Review by wojtek-kalicinski — CHANGES_REQUESTED** [review]
I'm only doing a sanity check, not fully reviewing code, as I never really worked on this library, I only helped open source it. So I trust the code works, and that you had your own round of reviews.

File: java/com/google/devrel/gmscore/tools/apk/arsc/Chunk.java (hunk 1 above)
Comment (ricowind) [reply java/com/google/devrel/gmscore/tools/apk/arsc/Chunk.java]: changed to aosp link

**Comment by ricowind** [comment]
> I'm only doing a sanity check, not fully reviewing code, as I never really worked on this library, I only helped open source it. So I trust the code works, and that you had your own round of reviews.

All of this code has been reviewed internally

**Pull request merged**